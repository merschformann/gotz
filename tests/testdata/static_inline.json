{
  "config_version": "1.0",
  "timezones": [
    {
      "Name": "New York",
      "TZ": "America/New_York"
    },
    {
      "Name": "Berlin",
      "TZ": "Europe/Berlin"
    },
    {
      "Name": "Shanghai",
      "TZ": "Asia/Shanghai"
    },
    {
      "Name": "Sydney",
      "TZ": "Australia/Sydney"
    }
  ],
  "style": {
    "symbols": "rectangles",
    "colorize": false,
    "day_segments": {
      "morning": 6,
      "day": 8,
      "evening": 18,
      "night": 22
    },
    "coloring": {
      "StaticColorMorning": "red",
      "StaticColorDay": "yellow",
      "StaticColorEvening": "red",
      "StaticColorNight": "blue",
      "StaticColorForeground": "",
      "DynamicColorMorning": "red",
      "DynamicColorDay": "yellow",
      "DynamicColorEvening": "red",
      "DynamicColorNight": "blue",
      "DynamicColorForeground": "",
      "DynamicColorBackground": ""
    }
  },
  "tics": false,
  "stretch": true,
  "inline": true,
  "hours12": false,
  "live": false
}
