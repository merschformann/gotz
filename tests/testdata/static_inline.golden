                                                now v 16:00
Local   : Sat 24 Aug 1985 14:00        ▒▒▒██████████|██████▒▒▒▒▒▒▒      
New York: Sat 24 Aug 1985 10:00               ▒▒▒███|█████████████▒▒▒▒▒▒
Berlin  : Sat 24 Aug 1985 16:00     ▒▒▒█████████████|███▒▒▒▒▒▒          
Shanghai: Sat 24 Aug 1985 22:00 ██████████████▒▒▒▒▒▒|             ▒▒▒███
Sydney  : Sun 25 Aug 1985 00:00 ██████████▒▒▒▒▒▒▒   |         ▒▒▒▒██████
